### DISCUSSION TITLE
The Roman counterpart of the god Poseidon gives his name to which planet?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#706] [Galaxy Trivia] [answered]
*answer with its astronomical symbol*

**Comment by peterbaccaro** [comment]
**♆** for Neptune

**Comment by SANKARAMDAS** [comment, marked as answer]
Neptune ♆ , Latin Neptunus, in Roman religion, originally the god of fresh water  ♆ ;

**Reply by ParanoidUser** [reply]
>  originally the god of fresh water  

Looks like executives finally recognized Neptune's hard work and promoted him to head of the Sea Department at some point. 😄

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [🥉 Galaxy Brain](https://github.com/SANKARAMDAS?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Silver 🥈 badge next by answering 16 questions. Don't forget to post your own questions too! 😊🎉

**Reply by SANKARAMDAS** [reply]
Thanks, ParanoïdUser

**Comment by thuongtruong1009** [comment]
The Roman counterpart of the god Poseidon is Neptune. Its astronomical symbol is ♆ (a trident), representing the god's weapon often associated with Poseidon/Neptune in mythology.
